tests/model_explainability/trustyai_service/conftest.py

```diff
@@ -386,3 +389,69 @@ def gaussian_credit_model(
                 runtime_name=mlserver_runtime.name,
             )
             yield isvc
+
+
+@pytest.fixture(scope="class")
+def isvc_getter_service_account(
+    admin_client: DynamicClient, model_namespace: Namespace
+) -> Generator[ServiceAccount, Any, Any]:
+    with ServiceAccount(client=admin_client, name=ISVC_GETTER, namespace=model_namespace.name) as sa:
+        yield sa
+
+
+@pytest.fixture(scope="class")
+def isvc_getter_role(admin_client: DynamicClient, model_namespace: Namespace) -> Generator[Role, Any, Any]:
+    with Role(
+        client=admin_client,
+        name=ISVC_GETTER,
+        namespace=model_namespace.name,
+        rules=[
+            {
+                "apiGroups": ["serving.kserve.io"],
+                "resources": ["inferenceservices"],
+                "verbs": ["get", "list", "watch"],
+            }
+        ],
+    ) as role:
+        yield role
+
+
+@pytest.fixture(scope="class")
+def isvc_getter_role_binding(
+    admin_client: DynamicClient,
+    model_namespace: Namespace,
+    isvc_getter_role: Role,
+    isvc_getter_service_account: ServiceAccount,
+) -> Generator[RoleBinding, Any, Any]:
+    with RoleBinding(
+        client=admin_client,
+        name=ISVC_GETTER,
+        namespace=model_namespace.name,
+        subjects_kind="ServiceAccount",
+        subjects_name=isvc_getter_service_account.name,
+        role_ref_kind="Role",
+        role_ref_name=isvc_getter_role.name,
+    ) as rb:
+        yield rb
+
+
+@pytest.fixture(scope="class")
+def isvc_getter_token_secret(
+    admin_client: DynamicClient,
+    model_namespace: Namespace,
+    isvc_getter_service_account: ServiceAccount,
+    isvc_getter_role_binding: RoleBinding,
+) -> Generator[Secret, Any, Any]:
+    with Secret(
+        client=admin_client,
+        name="sa-token",
+        namespace=model_namespace.name,
+        annotations={"kubernetes.io/service-account.name": ISVC_GETTER},
+        type="kubernetes.io/service-account-token",
+    ) as secret:
+        yield secret
```

Comment:
I'm not sure what this Secret is being used for, even in the call to `create_inference_token` it is not needed and that returns the token directly